variables.tf

```diff
@@ -265,13 +265,13 @@ variable "hms_docker_version" {
 variable "hms_rw_ecs_task_count" {
   description = "Desired ECS task count of the read/write Hive Metastore service."
   type        = "string"
-  default     = "2"
+  default     = "3"
```

Comment:
Was someone having issues with only 2 instances of the ECS containers?

Reply:
we saw issues when waggle-dance and metastores end up in different physical AZs , this is to increase chance of both being in same physical AZ